### ISSUE TITLE
Have you verified that your web app is accessible to users with disabilities by testing with screen readers, keyboard navigation, and other assistive technologies? Does your web app adhere to accessibility standards and guidelines?

### ISSUE THREAD

**Issue by amfairley** [amfairley/memorycards#17]
- [x] Link to the WAVE validator results in the README
- [x] Explain the feedback from the partially sighted user who uses a screen reader
- [x] Link to where you tested with just mouse/keyboard
- [x] Add a link to accessibility standards and guidelines (https://www.w3.org/WAI/WCAG22/quickref/)
- [x] Explain why it does not meet specific guidelines if required

**Comment by amfairley** [comment]
**Link to the WAVE validator results in the README**
- Results linked

**Explain the feedback from the partially sighted user who uses a screen reader**
- Provided in the user testing section

**Link to where you tested with just mouse/keyboard**
- This is present in the PATCHNOTES.md accessibility section

**Add a link to accessibility standards and guidelines (https://www.w3.org/WAI/WCAG22/quickref/)**
- Link added to the top of the accessibility section

**Explain why it does not meet specific guidelines if required**
- Meets all guidelines
